### ISSUE TITLE
[FR]: clearer instructions on reviewing submissions

### ISSUE THREAD

**Issue by Martinligabue** [waldo-vision/waldo#188]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### General Idea

It would be helpful to have some faqs about reviewing

### Problem

[In this section](https://waldo.vision/submissions/review) there is not much information on how to accept or reject a submission, nor is there a place to discuss it. For example, I had the question "Does this clip show gameplay from Counter Strike: Global Offensive" on a Counter Strike 2 clip and I don't know if I should say yes because they are merged or if there is a specific category.
Or what if a clip is almost all gameplay, but in some moments the clip is about other games or the desktop?

### Projects

Website

### Any Alternatives?

n/a

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Martinligabue** [comment]
yep, for that I choose to "watch on youtube" and decide on that video, but I'm not sure it's the right way

**Comment by ceriddenn** [comment]
I will add some more clarification to the current guidelines on what you suggested to the guidelines Rican made (haven’t been made public yet) and I will post them here or have someone post them in the discord.
